qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -64,24 +67,65 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
             # if we don't delete, then this value clashes with our quantum kernel
             del kwargs["kernel"]
 
-        self._quantum_kernel = quantum_kernel if quantum_kernel else QuantumKernel()
-
         if "random_state" not in kwargs:
             kwargs["random_state"] = algorithm_globals.random_seed
 
-        super().__init__(kernel=self._quantum_kernel.evaluate, *args, **kwargs)
+        # Class Fields
+        self._quantum_kernel = quantum_kernel or QuantumKernel()
```

Comment:
Ideally when checking against None it should be explicitly done since objects can override __bool__ and return True/False for non None entities.